dosma/core/med_volume.py

```diff
@@ -147,12 +151,14 @@ class MedicalVolume(NDArrayOperatorsMixin):
         affine (array-like): 4x4 array corresponding to affine matrix transform in RAS+ coordinates.
             Must be on cpu (i.e. no ``cupy.ndarray``).
         headers (array-like[pydicom.FileDataset]): Headers for DICOM files.
+        lazy (bool, optional): Convert the volume to a NumPy or CuPy array.
```

Comment:
The term `lazy` indicates that the volume should be lazily mapped - i.e. *not converted* to numpy/cupy array.

Reply:
I've updated the docs on this one.